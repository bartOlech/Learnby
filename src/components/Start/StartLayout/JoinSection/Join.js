import React from 'react';
import styled from 'styled-components';
import CreateAccountBtn from './CreateAccountBtn';
import { FontStyle } from '../../../../assets/style/style';

const Container = styled.div`
    width: 100%;
    height: 100%;
    margin-top: -200px;
`
const TittleText = styled.h2`
    font-family: ${FontStyle.family};
    font-size: 1.8em;
    font-weight: 500;
    color: ${FontStyle.color};
    text-align: center;
    @media (min-width: 1000px) {
        font-size: 2.2em;
        margin-bottom: 0px;
    }
`

const Join = () => {
    return ( 
        <Container>
            <TittleText>Uczmy siÄ™ razem!</TittleText>
            <CreateAccountBtn></CreateAccountBtn>
        </Container> 
    )
}

export default Join;