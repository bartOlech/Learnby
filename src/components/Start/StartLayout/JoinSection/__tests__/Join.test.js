import React from 'react';
import { render } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";
import Join from '../Join';
import { BrowserRouter as Router, Link } from 'react-router-dom';

test('render', () => {
    const { asFragment } = render(<Router><Join/></Router>);
    expect(asFragment()).toMatchSnapshot();
})

