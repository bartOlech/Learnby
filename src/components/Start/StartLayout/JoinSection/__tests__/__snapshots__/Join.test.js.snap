// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<DocumentFragment>
  <div
    class="sc-htpNat iwuHyY"
  >
    <h2
      class="sc-bxivhb iBKOoG"
    >
      Uczmy się razem!
    </h2>
    <div
      class="sc-bdVaJa cBJcMp"
    >
      <a
        href="/login"
      >
        <button
          class="sc-bwzfXH kOWHHY"
        >
          Załóż konto
        </button>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
